### PR TITLE
[Backport 2.x] Fix typo in recipient (#287)

### PR DESCRIPTION
**Pull request by cwperks**
Backport #287 to 2.x

### CODE REVIEW

**Review by yuye-aws — APPROVED** [review]

**Comment by yuye-aws** [comment]
@cwperks Nice work! Let's wait for the CI workflow before merging this PR.

**Review by Hailong-am — APPROVED** [review]

**Pull request merged**
